src/components/TimeseriesDataExport/TimeseriesDataExport.tsx

```diff
@@ -138,7 +148,7 @@ const TimeseriesDataExportFC: FC<TimeseriesDataExportFormProps> = (
     event: SyntheticEvent<HTMLInputElement>
   ) => {
     const [start, end] = form.getFieldValue('range');
-    const seriesNumber = timeseriesIds.length;
+    const seriesNumber = timeseriesNumber;
```

Comment:
this is redundant, cause you can just use `timeseriesNumber`

Reply:
thanks, fixed.